hudi-common/src/main/java/org/apache/hudi/common/table/read/BufferedRecordSerializer.java

```diff
@@ -66,6 +67,8 @@ <T> byte[] serialize(BufferedRecord<T> record, RecordSerializer<T> recordSeriali
           output.writeVarInt(record.getSchemaId(), true);
         }
         output.writeBoolean(record.isDelete());
+        HoodieOperation operation = record.getHoodieOperation();
```

Comment:
Can you update `TestBufferedRecordSerializer` to include a case where the operation is non-null?

Reply:
ok